src/com/jpexs/decompiler/flash/gui/locales/EasyPanel_zh.properties

```diff
@@ -33,7 +33,7 @@ item.movieClip = \u5f71\u7247\u526a\u8f91
 item.button = \u6309\u94ae
 item.sound = \u58f0\u97f3
 item.video = \u89c6\u9891
-item.uknown = \u672a\u77e5
+item.unknown = \u672a\u77e5
```

Comment:
I usually don't change translation files (I can, but it's a coin flip whether repositories want me to...)